<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_HRate</class>
 <widget class="QDialog" name="Dialog_HRate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>546</width>
     <height>234</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_22">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBoxPlayer">
        <property name="title">
         <string>Jugador 01</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelCardio">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxOnline">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OnLine</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxPlayer_2">
        <property name="title">
         <string>Jugador 02</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLabel" name="labelCardio_2">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxOnline_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OnLine</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxPlayer_3">
        <property name="title">
         <string>Jugador 03</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="labelCardio_3">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxOnline_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OnLine</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxPlayer_4">
        <property name="title">
         <string>Jugador 04</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QLabel" name="labelCardio_4">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxOnline_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OnLine</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxPlayer_5">
        <property name="title">
         <string>Jugador 05</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <widget class="QLabel" name="labelCardio_5">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxOnline_5">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OnLine</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxPlayer_6">
        <property name="title">
         <string>Jugador 06</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <widget class="QLabel" name="labelCardio_6">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxOnline_6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OnLine</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBoxPlayer_7">
        <property name="title">
         <string>Jugador 07</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <item>
          <widget class="QLabel" name="labelCardio_7">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxOnline_7">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OnLine</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxPlayer_8">
        <property name="title">
         <string>Jugador 08</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <widget class="QLabel" name="labelCardio_8">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxOnline_8">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OnLine</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxPlayer_9">
        <property name="title">
         <string>Jugador 09</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <widget class="QLabel" name="labelCardio_9">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxOnline_9">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OnLine</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxPlayer_10">
        <property name="title">
         <string>Jugador 10</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <item>
          <widget class="QLabel" name="labelCardio_10">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxOnline_10">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OnLine</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxPlayer_11">
        <property name="title">
         <string>Jugador 11</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_21">
         <item>
          <widget class="QLabel" name="labelCardio_11">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxOnline_11">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OnLine</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDesconectar">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Desconectar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
